.github/workflows/azure_e2e_release_promotion_tests.yml

```diff
@@ -0,0 +1,205 @@
+name: Azure E2E Release Promotion Tests
+'on':
+  push:
+    branches:
+      - aj-1436-e2e-staging
+  workflow_dispatch:
+    inputs:
+      testEnv:
+        description: >-
+          Environment in which tests should be run. Currently only runs on
+          staging
+      tenant-id:
+        description: 'Azure tenant ID. This is a shared identifier used by all Apps deployed for Azure End-to-End (E2E) tests. By default, it is set to the current Azure tenant ID.'
+        required: false
+        type: string
+        default: 'fad90753-2022-4456-9b0a-c7e5b934e408'
+      subscription-id:
+        description: 'Azure subscription ID. This is a shared identifier used by all Apps deployed for Azure End-to-End (E2E) tests. By default, it is set to the current Azure subscription ID.'
+        required: false
+        type: string
+        default: 'f557c728-871d-408c-a28b-eb6b2141a087'
+      mrg-id:
+        description: 'Azure Managed Resource Group name. This is a shared identifier used by all Apps deployed for Azure End-to-End (E2E) tests. By default, it is set to the current Azure MRG name.'
+        required: false
+        type: string
+        default: 'e2e-8n6xqg'
```

Comment:
Are these used anywhere? 🤔

Reply:
They were/will be!  I temporarily removed them the places they're used just to see if I could get some data on whether or not I broke things with [this commit](https://github.com/DataBiosphere/leonardo/pull/4014/commits/b60b450a65d1ef5c61ec0d2ef268f9139802ed4d).  Planning to un-revert those changes